### DISCUSSION TITLE
Question about using the handle returned by MXPredReshape

### DISCUSSION THREAD

**Discussion by pango99** [apache/incubator-mxnet#16867] [Q&A] [answered]
Hi, I'm using mxnet C API and ArcFace model to do face feature encoding, I found the API MXPredReshape() can change the input shape of net,and return a new predictor handle with the new shape,so after call the api,how about the old predictor handle with old shape? Can I use the old handle continually? 

I had do a test and found the old & new handle all can be used,but I'm not sure what I do is right, so I post the issue to confirm;

 I plan to use MXPredReshape() to create a net that has more than one input shapes(MXPredCreate() create main handle with a fixed shape, and MXPredReshape() create some handles with different shape,every shape has different batch parameter,chw is the same), is my idea achievable?

**Comment by szha** [comment, marked as answer]
Yes, what you described is correct and intended.
